### DISCUSSION TITLE
A Linux version of the engine (Stride)

### DISCUSSION THREAD

**Discussion by GATETECAT4** [stride3d/stride#1893] [Ideas]
I am very interested in Stride Game Engine, and I would like to try it, but Stride only has support for Windows and MacOS (if I'm not wrong), not Linux as you can imagine. So a Linux version doesn't sound so bad.

**Comment by Eideren** [comment]
The editor only supports windows right now, you can build games from windows to ship to linux or macos, there is also a code-only api to build games directly from linux or macos.
We're looking into porting the editor to other platforms, but this isn't going to come out over night. You can see some of the discussion around this over here: https://github.com/stride3d/stride/issues/8

**Comment by MostHated** [comment]
I definitely would like to throw in my vote for a linux version of the editor.

**Comment by JeromyWalsh** [comment]
Adding a Linux version of the Game Studio is a multi-faceted objective with a few of us tackling different components. There's the #linux channel on the Stride Discord if you want to participate there.
